src/components/ExpandableInfo.tsx

```diff
@@ -1,216 +1,155 @@
-      <Content>
-        {image && <GatsbyImage image={image} alt="" />}
-        <TitleContent>
-          <Question>
-            <Header>
-              <Title>{title}</Title>
-            </Header>
-            <TextPreview>{contentPreview}</TextPreview>
-          </Question>
-        </TitleContent>
-        {!forceOpen && (
-          <ButtonContainer
-            variants={chevronFlip}
-            animate={isVisible ? "expanded" : "collapsed"}
-            initial={false}
-            onClick={() => setIsVisible(!isVisible)}
-          >
-            <Icon name="chevronDown" size="36" />
-          </ButtonContainer>
-        )}
-      </Content>
-      <motion.div
-        variants={expandCollapse}
-        animate={isVisible ? "expanded" : "collapsed"}
-        initial={false}
+    }
+
+    const animateToggle = isOpen ? "expanded" : "collapsed"
+
+    return (
+      <VStack
+        border="1px solid"
+        borderColor="border"
+        borderRadius="2px"
+        p="1.5rem"
+        mb="1rem"
+        spacing="0"
+        background={background ?? "background"}
+        position="relative"
+        _hover={{
+          "& img": {
+            transform: "scale(1.08)",
+            transition: "transform 0.1s",
+          },
+        }}
+        ref={ref}
+        {...rest}
       >
-        <motion.div
-          variants={showHide}
-          animate={isVisible ? "expanded" : "collapsed"}
+        <Stack
+          justify="space-between"
+          gap={{ base: "2rem", md: "3rem" }}
+          flexDirection={{ base: "column", md: "row" }}
+          width="full"
+        >
+          {image && <GatsbyImage image={image} alt="" />}
+          <HStack gap="3rem" width="full">
+            <Box mr="1rem">
+              <HStack
+                width="full"
+                m="1rem 0"
+                sx={{
+                  img: {
+                    mr: "1.5rem",
+                  },
+                }}
+              >
+                <Heading mt="0rem" mb="0.5rem" size="md" fontWeight="600">
+                  {title}
+                </Heading>
+              </HStack>
+              <Text color="text200" mb="0">
+                {contentPreview}
+              </Text>
+            </Box>
+          </HStack>
+          {!forceOpen && (
+            <Center
+              as={motion.div}
+              variants={chevronFlip}
+              animate={animateToggle}
+              initial={false}
+              {...getButtonProps()}
+              width={{ base: "full", md: "5rem" }}
+              minHeight={{ base: "full", md: "10rem" }}
+              cursor="pointer"
+              _hover={{
+                "& svg": {
+                  transform: "scale(1.25)",
+                  transition: "transform 0.1s",
+                },
+              }}
+            >
+              <Icon as={MdExpandMore} boxSize="initial" size="36" />
+            </Center>
+          )}
+        </Stack>
+        <Collapse
```

Comment:
Transition effect simplified to the single `Collapse` component, as all the fade-in/out effects are apart of the collapse transition.